views/schedule.mako

```diff
@@ -77,6 +77,7 @@
     </div>
 </div>
 
+<div class="horizontal-scroll">
```

Comment:
It's outside the if so it will encompass any of the layouts.  They all have issues that having a horizontal scroll on mobile view will fix.